tests/rt_ccpp_HAFS.conf

```diff
@@ -0,0 +1,11 @@
+# Compile with CCPP - dynamic mode
+COMPILE | CCPP=Y REPRO=Y                  | standard    | hera.intel     |             |
+# Run tests, input data currently only available on Theia
+#RUN    | fv3_ccpp_ferhires               | standard    |                |             |
+RUN    | fv3_ccpp_regional_c768_FA_update_moist   | standard    |                |             |
```

Comment:
So, for the merge, the new fv3_ccpp_regional_c768_FA_update_moist test should be added to a different rt*.conf? The CCPP section of rt.conf?

Reply:
This should be added to rt_ccpp_dtc.conf for the time being, no new rt_ccpp_HAFS.conf please.